src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1727,6 +1727,110 @@ static SvgStyleGradient* _cloneGradient(SvgStyleGradient* from)
 }
 
 
+static void _styleInherit(SvgStyleProperty* child, const SvgStyleProperty* parent)
```

Comment:
I moved this func from below, without any changes.

Of course, I can unify the `_styleInherit` and the `_styleCopy` functions, by adding a bool as an additional arg:
```
static void _styleInherit(SvgStyleProperty* style1, const SvgStyleProperty* style2, bool b)
```
with the comparisons inside looking like:
```
if (((int)style1->fill.flags & (int)SvgFillFlags::Paint) == b) {
```

but please notice how hard it became to read and understand. 
If `b` is `false` then this comparison behaves as in  `_styleInherit`. 
if `b` is `true` then it's like  `_styleCopy`, but I have to call it `_styleCopy(from->style, to->style);` (instead of `_styleCopy(to->style, from->style)`), that's why I'd have to change the args names parent/child or from/to to style1/style2.

Reply:
I know there is a lot of duplicate code. As you said, the code(`bool b...`) you presented is difficult to understand and complex.
Because inherit and copy have different if conditions, I think it's okay to have separate method.